cloudnetpy/products/ier.py

```diff
@@ -0,0 +1,345 @@
+    freq = utils.get_frequency(ier.wl_band)
+    coeffs = ier.coeffs
+    factor = round((coeffs[0] / 0.93) * 1000) / 1000
+    attributes["ier"] = attributes["ier"]._replace(
+        comment=f"This variable was calculated from the {freq}-GHz radar reflectivity \n"
+        f"factor after correction for gaseous attenuation, and temperature taken from \n"
+        f"a forecast model, using the following empirical formula: \n"
+        rf"log10($\alpha$[g m-3]) = freq{coeffs[1]}Z[dBZ]T[degC] + "
+                f"{coeffs[3]}Z[dBZ] + {coeffs[2]}T[degC] + {coeffs[4]}.\n"
+        "In this formula $\\mathregular{\\alpha}$ is taken to be defined such that all \n"
+        "frequencies of radar would measure the same Z in Rayleigh scattering ice.\n"
+        "However, the radar is more likely to have been calibrated such that all \n"
+        "frequencies would measure the same Z in Rayleigh scattering\n"
+        "liquid cloud at 0 degrees C. The measured Z is therefore multiplied by \n"
+        f"|K(liquid,0degC,{freq}GHz)|^2/0.93 = {factor}  before applying this formula.\n"
+        "The formula has been used where the \"categorization\" data has diagnosed \n"
+        "that the radar echo is due to ice, but note that in some cases\n"
+        "supercooled drizzle will erroneously be identified as ice. Missing data \n"
+        "indicates either that ice cloud was present but it was only\n"
+        "detected by the lidar so its ice water content could not be estimated, or \n"
+        "that there was rain below the ice associated with uncertain\n"
+        "attenuation of the reflectivities in the ice.\n"
+        "Note that where microwave radiometer liquid water path was available it was \n"
+        "used to correct the radar for liquid attenuation when liquid\n"
+        "cloud occurred below the ice; this is indicated a value of 3 in the \n"
+        "iwc_retrieval_status variable.  There is some uncertainty in this\n"
+        "prodedure which is reflected by an increase in the associated values in the \n"
+        "iwc_error variable. When microwave radiometer data were not available and \n"
+        "liquid cloud occurred below the ice, the retrieval was still performed but its\n"
+        "reliability is questionable due to the uncorrected liquid water attenuation.\n"
+        "This is indicated by a value of 2 in the iwc_retrieval_status\n"
+        "variable, and an increase in the value of the iwc_error variable"
+    )
+    return attributes
+
+
+COMMENTS = {
+    "ier": (
+        "Ice effective radius Hogen et al. 2006."
+    ),
+    "ier_retrieval_status_comment": (
+        "This variable describes whether a retrieval was performed for each\n"
+        "pixel, and its associated quality, in the form of 8 different classes.\n"
+        "The classes are defined in the definition and long_definition attributes. \n"
+        "The most reliable retrieval is that without any rain or liquid\n",
```

Comment:
There's a misplaced comma here and fixing this should fix the type error:
```suggestion
        "The most reliable retrieval is that without any rain or liquid\n"
```